pom.xml

```diff
@@ -1486,6 +1486,7 @@
       <exclude>**/patchprocess/**</exclude>
       <exclude>**/metastore_db/**</exclude>
       <exclude>**/test/resources/**/*.ldif</exclude>
+      <exclude>.vscode/**</exclude>
```

Comment:
Since this has nothing to do with JSON formatting in beeline, save it for another JIRA.